### PR TITLE
Implement login to AuthenticationService and findByEmail method in U…

### PR DESCRIPTION
**Pull request by VolodymyrKachan**
…serService

### CODE REVIEW

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+            UserService userService = new UserService();
+            User user = userService.findByEmail(email);
+            return user != null && email.equals(user.getEmail()) && password.equals(user.getPassword());
+        }
     }
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/service/AuthenticationService.java:21]: does your code compile?

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+            UserService userService = new UserService();
+            User user = userService.findByEmail(email);
+            return user != null && email.equals(user.getEmail()) && password.equals(user.getPassword());
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/service/AuthenticationService.java:19]:
```suggestion
            return user != null && password.equals(user.getPassword());
```
you already found user by email, so they are equal

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/service/AuthenticationService.java [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+        return user != null && password.equals(user.getPassword());
+        }
     }
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/service/AuthenticationService.java:21]: code style failed - run mvn clean package and fix mistakes

File: src/main/java/mate/academy/service/UserService.java [hunk 4]
```diff
@@ -3,18 +3,24 @@
...
 
     /**
      * Find user by email. All users are stored in <code>private static final User[] users</code>
+     *
      * @param email - the input parameter
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+        for (User userEmail : users) {
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/service/UserService.java:19]:
```suggestion
        for (User user : users) {
```

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by Sofasmile — APPROVED** [review]

**Pull request closed without merging**